pkg/server/apiextensions.go

```diff
@@ -92,6 +92,12 @@ func newSystemCRDProvider(
 			clusters.ToClusterAwareKey(SystemCRDLogicalCluster, "workspaces.tenancy.kcp.dev"),
 		),
 		universalCRDs: sets.NewString(
+			clusters.ToClusterAwareKey(SystemCRDLogicalCluster, "clusterworkspaces.tenancy.kcp.dev"),
+			clusters.ToClusterAwareKey(SystemCRDLogicalCluster, "clusterworkspacetypes.tenancy.kcp.dev"),
```

Comment:
nit: does this mean we want them everywhere? Then I would expect this to be outside of the swtch statement.